src/elements/controls/graphics.ts

```diff
@@ -0,0 +1,41 @@
+import { BuildOutput } from "@src/windows/buildOutput";
+import { ParentControl } from "@src/windows/parentControl";
+import { WidgetCreator } from "@src/windows/widgets/widgetCreator";
+import { ElementParams } from "../elementParams";
+import { AbsolutePosition } from "../layouts/absolute/absolutePosition";
+import { FlexiblePosition } from "../layouts/flexible/flexiblePosition";
+import { Control } from "./control";
+
+/**
+ * The parameters for configuring the custom graphics drawing widget.
+ */
+export interface GraphicsParams extends ElementParams {
```

Comment:
Could you change the bracket style in this file to [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Notable_styles), like how it is used in the rest of the project?